src/view/UserphraseView.cpp

```diff
@@ -75,6 +79,12 @@ void UserphraseView::addPhrase(int result)
     }
 
     emit model()->add(phrase, bopomofo);
+    
+    if (dialogType_ == DIALOG_MODIFY) {
+	if(model()->getAddResult() <= 0) {
+            emit model()->add(originalPhrase, originalBopomofo);
```

Comment:
Do the "original frequency" means how ofter the user type them?
Well, I didn't consider it, where does the editor deal with it?
May be I can take it a look.